### PR TITLE
RAIL-3602 ci configuration

### PR DESCRIPTION
**Pull request by plavemar**
- e2e cypress tests run in docker
- e2e cypress tests run in ci
- GitHub report script

JIRA: RAIL-3602, RAIL-3605

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4800/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6440/ : SUCCESS in 14m 33s

File: libs/sdk-ui-tests-e2e/package.json [hunk 1]
```diff
@@ -143,6 +144,7 @@
         "source-map-loader": "^2.0.2",
         "speed-measure-webpack-plugin": "^1.5.0",
         "style-loader": "^1.2.1",
+        "styled-jsx": "~3.2.5",
```
Comment (no23reason) [inline libs/sdk-ui-tests-e2e/package.json:147]: Please use ^

File: libs/sdk-ui-tests-e2e/package.json [hunk 2]
```diff
@@ -84,7 +84,8 @@
         "react-router-dom": "^5.2.0",
         "react-router-use-location-state": "~2.5.0",
         "tslib": "^2.0.0",
-        "wait-on": "^6.0.0"
+        "wait-on": "^6.0.0",
+        "xml2js": "~0.4.23"
```
Comment (no23reason) [inline libs/sdk-ui-tests-e2e/package.json:88]: Please use ^

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4803/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6444/ : FAILURE in 45s

**Review by no23reason — APPROVED** [review]
LGTM

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1626

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1360/ : FAILURE in 3m 55s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1636/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/420/ : SUCCESS in 13m 39s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6446/ : SUCCESS in 19m 04s

**Pull request merged**